src/main/scala/io/flow/log/Rollbar.scala

```diff
@@ -139,13 +140,27 @@ object RollbarProvider {
       }
     }
 
+    val configurationOverrider = new Transformer {
+      override def transform(data: Data): Data = {
+        Option(data.getCustom)
+          .flatMap(custom => Option(custom.get(RollbarLogger.Keys.Environment)))
+          .flatMap(env => Option.when(!env.toString.isBlank)(env.toString))
+          .fold(ifEmpty = data) { environment =>
+            new Data.Builder(data)
+              .environment(environment)
+              .build()
+          }
+      }
+    }
```

Comment:
Note you can use SAM here

```suggestion
    val configurationOverrider: Transformer = {
      data: Data =>
        Option(data.getCustom)
          .flatMap(c => Option(c.get(RollbarLogger.Keys.Environment)))
          .flatMap(e => Try(e.asInstanceOf[String]).toOption) match {
          case Some(e) if !e.isBlank => new Data.Builder(data).environment(e).build()
          case _ => data
        }
    }
```